src/jwplayer/players/shared/JwPlayerWrapper.tsx

```diff
@@ -46,9 +46,15 @@ const JwPlayerWrapper: React.FC<JwPlayerWrapperProps> = ({
 	useEffect(() => {
 		if (shouldLoadSponsoredContentList) {
 			const retrieveSponsoredVideo = async () => {
```

Comment:
Why defining a function inside the `useEffect` hook? The hook could be leaner without it.

Reply:
I agree. We could extract it and even unit-test it. I'll fill a follow-up ticket, so we don't slow down the process as it's needed to be live on Monday 🤷‍♂️

Reply:
https://fandom.atlassian.net/browse/VIDEO-580